blog/2023/2023-09-21-supercharge-your-repository-with-code-owners-4clg.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Supercharge your Repository with Code Owners"
+tags: []
+authors: nickytonline
+slug: 2023-09-21-supercharge-your-repository-with-code-owners-4clg
```

Comment:
I see that your slugs aren't prefixed by a date @BekahHW. Is the convention like so?

file path: {year_folder}/{date}-{slug}

and front matter:

slug: {slug}